config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-
+  root :to => 'pages#homepage'
 
 
   get "book_in_company/home", :to => "book_in_company#home"
```

Comment:
What about handling the different static pages for the public, incompany and sponsor pages from your `PagesController` as well?

If you do that, these 3 routes would be changed into this:

```ruby
  get '/public', to: 'pages#public'
  get '/incompany', to: 'pages#incompany'
  get '/sponsor', to: 'pages#sponsor'
```

And then in your `PagesController` you just add those methods:

```ruby
class PagesController < ApplicationController
  def homepage
  end  

  def public
  end  

  def incompany
  end  

  def sponsor
  end  
end
```

You just need to create corresponing view files.